### ISSUE TITLE
[CAL-3954] Hard to Focus on Text Area to enter User About description 

### ISSUE THREAD

**Issue by nithingodugu24** [calcom/cal.com#11378]
### Is your proposal related to a problem?

It is very hard to focus on the text area to enter the description in About section. And also the formatting options are just above the text area. So we i can't able to find the focus of text area.

### Describe the solution you'd like

Try to increase the text area height so that we can easily focus to text area. And also shift the formatting options to right of text area .

<sub>[CAL-3954](https://linear.app/calcom/issue/CAL-3954/hard-to-focus-on-text-area-to-enter-user-about-description)</sub>

**Comment by MeenuyD** [comment]
Hello @nithingodugu24 On which page

**Comment by shivamhwp** [comment]
i think @nithingodugu24 is talking about the profile page

**Comment by nithingodugu24** [comment]
@shivamhwp Yes, I am encountering this issue on the signup page as well as the profile edit page.

**Comment by PeerRich** [comment]
can you provide screenshots?

**Comment by ChaudharyPradip** [comment]
Hey, I looked at the issue. It seems that the component is working fine. when we add more text to the input it's height increases automatically. However, if we manually change the height of input using the controller on right bottom it doesn't change it's height according to text afterwards.

![WhatsApp Image 2023-09-15 at 20 51 54](https://github.com/calcom/cal.com/assets/88336529/043db57a-83ca-4866-bc63-8a745fa667d3)
![WhatsApp Image 2023-09-15 at 20 54 06](https://github.com/calcom/cal.com/assets/88336529/32d7fbd3-7bdb-4092-b60b-da543adac0a9)

This is happening because the input has resize option set to verticle. It can be solved by removing this CSS property.
I am submitting the PR by removing this. If this CSS property was intentional then please ignore this PR.

Also, please inform if the issue was not the one I am explaining @nithingodugu24

**Comment by B2-krunalrana** [comment]
attempt

**Comment by AbdulBasit2733** [comment]
@nithingodugu24  Hello Sir May i work at this issue.

**Comment by Nasirahamad88** [comment]
great

**Comment by Nasirahamad88** [comment]
use custom CSS to modify the appearance and layout of the text area in the About section

**Comment by wajidjamali** [comment]
good

**Comment by cholaraju** [comment]
nice

**Comment by mkos11** [comment]
Proposed Solution:

1. **Increase Text Area Height:**
   - Extend the height of the text area in the "About" section to make it more prominent and easier to focus on. This adjustment provides users with a larger space for entering and reviewing text.

2. **Reposition Formatting Options:**
   - Shift the formatting options toolbar to the right side of the text area. By relocating these options adjacent to the text input, users can more intuitively access formatting features while maintaining focus on the text.

Implementation Considerations:

1. **Responsive Design:**
   - Ensure that the proposed changes maintain responsiveness, allowing for an optimal user experience across various devices and screen sizes.

2. **User Feedback:**
   - Gather user feedback during and after the implementation to assess the effectiveness of the changes and make further adjustments if necessary.

3. **Accessibility:**
   - Verify that the modified layout remains accessible to users with different abilities and does not hinder usability for individuals using assistive technologies.

4. **Testing:**
   - Conduct thorough testing across browsers and devices to identify and address any potential issues that may arise from the adjustments.

By implementing these changes, the goal is to enhance the user experience, making it more convenient for users to focus on and interact with the text area while seamlessly accessing formatting options.

**Comment by AS-Danish** [comment]
The issue if fixed I think it doesn't seem as a problem

**Comment by karthikkatu** [comment]
Is the issue completed? If yes please close completed issues

**Comment by adityasoni2019** [comment]
is it still open?

**Comment by nikhildeshmukh77** [comment]
is it still open ?

**Comment by AnshDavhale** [comment]
> is it still open?

yes, it is still open

**Comment by talhafayyaz123** [comment]
is it still open?

**Comment by Jynx2004** [comment]
![Screenshot 2025-01-13 003719](https://github.com/user-attachments/assets/53e024c8-c165-47b6-93db-2547b5fa3760)

I think that adding a background color to the upper part of the text input would be a good thing . The users will be able to see the part where they can write and click on that easily.

**Comment by Jvald27** [comment]
still open? /assign please TEAM 6 (5 individuals)

**Comment by tech-dipesh** [comment]
is it stil open, can i start working now?

**Comment by TusharBhatt1** [comment]
The issue has been already fixed on fixed , thanks.

**Comment by TusharBhatt1** [comment]
Tried this - was able to write the description without any issue (may be the issue has been already) - however we can increase the text area height for better UX.
